src/logic/createTag.ts

```diff
@@ -8,28 +8,21 @@ export default function createTag({
   keyframes: Keyframes;
   animationName: string;
 }): {
-  styleTag: any;
+  styleTag: HTMLStyleElement;
   index: number;
 } {
-  let styleTag = document.querySelector('style[data-id=rsi]');
-  let index;
+  let styleTag = document.querySelector<HTMLStyleElement>('style[data-id=rsi]');
 
   if (!styleTag) {
     styleTag = document.createElement('style');
     styleTag.setAttribute('data-id', 'rsi');
     document.head.appendChild(styleTag);
   }
 
-  try {
-    // @ts-ignore
-    index = styleTag.sheet.cssRules.length;
-  } catch (e) {
-    index = 0;
-  }
+  const index = styleTag.sheet?.cssRules.length ?? 0;
```

Comment:
```suggestion
  const index = styleTag.sheet?.cssRules?.length ?? 0;
```

Avoid `undefined.length`